examples/workload_md.py

```diff
@@ -46,12 +46,18 @@ def prepare_md(wl):
             tbuffer  = tbuffer.replace('@temperature@', str(temp))
             tbuffer  = tbuffer.replace('@timesteps@',   str(nsteps))
 
-        fname = '%s/mdin.rep.%04d' % (inputs, i)
+       #fname = '%s/mdin.rep.%04d' % (inputs, i)
+       #mdp extension just for gromacs 
+       #change accordingly for other md executables
```

Comment:
Comments should start at the same position as code, and should have a white space after `#`.